src/napari/_qt/_tests/test_qt_viewer.py

```diff
@@ -1381,3 +1381,58 @@ def test_dask_cache(qt_viewer):
     mock_resize_dask_cache.assert_called_once_with(
         int(int(True) * initial_dask_cache * 1e9)
     )
+
+
+def test_viewer_drag_to_zoom(qt_viewer, qtbot):
+    """Test drag to zoom mouse binding."""
+    viewer = qt_viewer.viewer
```

Comment:
The `qt_viewer.viewer` is `ViewerModel` that could be accessed using `viewer_model`   fixture.